docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
 
-#### `How to access input parameters`
+#### `How to Access Input Parameters`
 
-Input parameters can be accessed as tokens directly in the actions (eg: [MyParam]).Keep in mind that the entities input parameters are expanded and you can access all the data of the entity (eg: [MyEntityParam:Property] or [ MyEntityParam :Id])
+Input parameters can be accessed as tokens directly in the actions (eg: [MyParam]). Keep in mind that the entities input parameters are expanded and you can access all the data of the entity (eg: `[MyEntityParam:Property]` or `[MyEntityParam:Id]`)
 
-#### `How to access other data through actions or tokens`
+#### `How to Access Other Data Through Actions or Tokens`
 
-Other data can be accessed with specific actions that read data from different sources. For example you can read other entities data based on a Server Request. This data came into the execution context without being based as an input parameter. Some actions that we can list in this category: Run SQL Query, Server Request, Read Entity Data also can come from Tokens (the module). By passing some of the input parameters to the token for example it can bring back a HTML code that can be used in Generate PDF action to obtain a PDF document.
+Other data can be accessed with specific actions that read data from different sources. For example, you can read other entities' data based on a Server Request. This data comes into the execution context without being specified as an input parameter. Some actions that we can list in this category include `Run SQL Query`, `Server Request`, and `Read Entity`. Data also can come from Plant an App Tokens. By passing some of the input parameters to the token, for example, it can bring back HTML code that can be used in a `Generate PDF` action to obtain a PDF document.
+
+#### `Importing an Action Configuration`
+
+You may also import an already configured action into a workflow to save time. To import an action you need to locate and select the desired action and then click the `Export` button. Copy the JSON presented in the popup. Then, in the workflow in which the action will be imported, click the `Import` button. The first time you perform this action, your browser will likely ask you to allow Plant an App access to your clipboard. The notification may look like this:
+
+<img src="/img/clipboard-access.png" alt="Clipboard Access Request" />
+
+If the content of the clipboard is not valid code **OR** if your browser has denied access to the clipboard, a pop-up titled `Import Actions` will appear. If access to the clipboard has been denied, the `Action or Array of Actions` box will be empty and you can paste your code into it. Otherwise, it will be populated with the invalid code text from the clipboard as seen below:
```

Comment:
```suggestion
If the content of the clipboard is not valid code **OR** if your browser has denied access to the clipboard, a pop-up titled `Import Actions` will appear. If access to the clipboard has been denied, the `Action or Array of Actions` box will be empty and you can paste your code into it. Otherwise, it will be populated with the invalid code text from the clipboard as seen below:
```